prow/cluster/horologium_deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       labels:
         app: horologium
     spec:
+      # serviceAccountName: "horologium" # Uncomment for use with RBAC
```

Comment:
I think you should be able to use this service account regardless of running with rbac or not so it may be worth to uncomment upfront and move the service accounts alongside the deployment manifests.